webpack/ForemanWebhooks/Routes/Webhooks/WebhooksIndexPage/Components/WebhooksTable/WebhooksTable.js

```diff
@@ -60,11 +60,11 @@ const WebhooksTable = ({
         style={{ marginBottom: -6 }}
       />
       <Pagination
-        viewType="list"
         itemCount={itemCount}
-        pagination={pagination}
-        onChange={fetchAndPush}
-        dropdownButtonId="webhooks-page-pagination-dropdown"
+        /* eslint-disable-next-line camelcase */
+        onChange={({ page, per_page }) =>
+          fetchAndPush({ page, perPage: per_page })
+        }
```

Comment:
this can remain a one line: `onChange={fetchAndPush}`
and in the action it's already passed as `per_page`:
https://github.com/theforeman/foreman_webhooks/blob/ea5f7e07a8ddd7ef8a437f159252c03436295e2e/webpack/ForemanWebhooks/Routes/Webhooks/WebhooksPageActions.js#L36

Reply:
Yeah, I thought that, but the reason why I do that explicitly is that `onChange` callback receives `per_page`, but `fetchAndPush` expects `perPage` and I didn't want to bring more changes.